### DISCUSSION TITLE
BOB integration. RunSQLSTM and library lists.

### DISCUSSION THREAD

**Discussion by beckhamrryyaann** [IBM/ibmi-tobi#205] [Q&A]
Let me reword my question so its not so vague.  I need BOB to respect the sql property in the iproj.json.  When building project i need Bob to run sql statements using the sql naming convention.  I do not want it walking down my library list looking for files.  The docs say sql naming is default but I don't think that is the case.  I have my iproj.json file explicitly defined to sql naming and it still uses system when it runs.   thanks

![image](https://user-images.githubusercontent.com/6051612/221015355-57f6cc6c-4ed6-4726-a44a-685d02ecca5a.png)

**Comment by beckhamrryyaann** [comment]
I think this also relates to #159 and #127   
I have the "sql" property specified in the iproj.json file and I am telling it to use SQL naming and setting my default Schema, but BOB is still looking through my entire library list looking for the object if it doesn't exist in my default schema....which I do not want. 

![image](https://user-images.githubusercontent.com/6051612/221199660-84f7d692-0393-4d42-bc75-1d7af6abbe49.png)

**Comment by edmundreinhardt** [comment]
Ryan, you are right that BOB does nto yet support that setting.  It is supported by Merlin at this point.  Is this a parameter we can set on RUNSQLSTM?
